### DISCUSSION TITLE
".men" is not a fully qualified domain name.

### DISCUSSION THREAD

**Discussion by rinorzk** [vercel/community#4733] [Help] [answered]
### Summary

I wanted to add my new domain that ends with ".men" and it's a valid TLD, I bought it on Namecheap, but Vercel complains about it and it won't let me add it. Can someone help me please I really need this 

### Steps to Reproduce

Try adding domains that have a more 'niche' TLD and not just the common .com, .io... it will complain as not being a fully qualified domain name.

**Comment by amyegan** [comment]
Did you follow the steps to [add and configure a domain](https://vercel.com/docs/projects/domains/add-a-domain#add-and-configure-domain), or did you try to [transfer the domain to Vercel](https://vercel.com/docs/projects/domains/transfer-your-domain#transfer-a-domain-to-vercel)?

**Reply by rinorzk** [reply]
I tried to [add and configure a domain](https://vercel.com/docs/projects/domains/add-a-domain#add-and-configure-domain), since I can't transfer to Vercel before the 60 days on a newly bought domain.

**Reply by amyegan** [reply, marked as answer]
I was able to start the process of adding a `.men` domain to an example project. I got to the message about needing an A record without it telling me it isn't a fully qualified domain. 

<img width="922" alt="settings page showing invalid configuration but no error about being fully qualified domain" src="https://github.com/vercel/community/assets/8247105/3e991743-2b37-44a4-ab96-5a1ac5dc3902">

Is it past this point that you see the fully qualified domain error message?

**Reply by rinorzk** [reply]
it might have been my bad, I copied the domain name, and it probably picked some hidden characters, it now works, thanks a lot!

**Reply by amyegan** [reply]
Glad to hear it's working now!
